src/components/v1/V1Project/index.tsx

```diff
@@ -19,7 +19,7 @@ import V1ProjectHeaderActions from './V1ProjectHeaderActions'
 import Rewards from './Rewards'
 import V1PayButton from './V1PayButton'
 
-const BalanceTimeline = lazy(() => import('./BalanceTimeline'))
```

Comment:
**question** now that `BalanceTimeline ` is unused, can we delete the component file?

Reply:
yes - we can, forgot to do that, thanks for the reminder! 🙏